mep/people/management/commands/export_creators.py

```diff
@@ -0,0 +1,41 @@
+"""
+Manage command to export creator data for use by others.
+
+Generates a CSV and JSON file including details on every creator
+database, with summary details and coordinates for associated addresses.
```

Comment:
looks like this is slightly out of date.

maybe worth briefly clarifying how we define creator?